### ISSUE TITLE
removing tag of linked library item

### ISSUE THREAD

**Issue by kutoman** [UnderwaterApps/overlap2d#163]
when I remove a tag from/ or add a tag to a linked library item, shouldn't be the original library item be affected (so that all clones are affected)? Or is it intended like in the current state (changes on tags only affect the individual items of the tags)?

**Comment by azakhary** [comment]
I am taking this one.

**Comment by azakhary** [comment]
hmm I am yet thinking if that's bug or not. you see when you drop a library item on the scene, it is "instance of that library item" which means it's "insides" are the same, but things like x, y coordinates for example are individual. if you move one, others don't move. So the tags.

**Comment by kutoman** [comment]
yes that is exactly what I'm wondering. But how could I edit the library item's tags otherwise

**Comment by kutoman** [comment]
Edit and create a new item is the current option

**Comment by azakhary** [comment]
there is another optino. you can unlink it, add tag, and link it again with previous name, it will overwrite.

**Comment by kutoman** [comment]
ah, sounds better. While testing it, I noticed a real small bug :D. When canceling the "New library Item" dialog, the link symbol keeps being orange

**Comment by kutoman** [comment]
There is another one. When overwriting a library item, the earlier clones not get updated. Maybe there should be:
1- Warning when user is about to overwrite an item
2a- refresh earlier clones
2b- break links of earlier clones
